benchmarks/benchmarks/stats.py

```diff
@@ -159,10 +159,16 @@ def setup(self):
         self.inp = np.random.rand(9999).reshape(3, 3333) * 200
         self.subbin_x_edges = np.arange(0, 200, dtype=np.float32)
         self.subbin_y_edges = np.arange(0, 200, dtype=np.float64)
-
-    def time_binned_statistic_2d_std(self):
-        stat = 'std'
-        binned_stat, _, _, binnumbers = stats.binned_statistic_2d(
-            self.inp[0], self.inp[1], values=self.inp[2], statistic=stat,
-            bins=[self.subbin_x_edges, self.subbin_y_edges],
-            expand_binnumbers=True)
+        self.ret = stats.binned_statistic_dd(
+            [self.inp[0], self.inp[1]], self.inp[2], statistic="std",
+            bins=[self.subbin_x_edges, self.subbin_y_edges])
+
+    def time_binned_statistic_dd_std(self):
+        stats.binned_statistic_dd(
+            [self.inp[0], self.inp[1]], self.inp[2], statistic="std",
+            bins=[self.subbin_x_edges, self.subbin_y_edges])
+
+    def time_binned_statistic_dd_std_reuse_bin(self):
+        stats.binned_statistic_dd(
+            [self.inp[0], self.inp[1]], self.inp[2], statistic="mean",
```

Comment:
I'm confused by the naming of the test and the statistic argument here. The test name states test is for std yet the statistic is `"mean"`? What is the reasoning?

Reply:
You are right. I just copy and paste the code from docstring, but we should use `std`. I changed it.